docs/en/sql-reference/statements/create/table.md

```diff
@@ -380,7 +380,7 @@ High compression levels are useful for asymmetric scenarios, like compress once,
 
 `DEFLATE_QPL` — [Deflate compression algorithm](https://github.com/intel/qpl) implemented by Intel® Query Processing Library. Some limitations apply:
 
-- DEFLATE_QPL is experimental and can only be used after setting configuration parameter `allow_experimental_codecs=1`.
+- DEFLATE_QPL is disabled by default and can only be used after set server setting `enable_deflate_qpl_codec` to be true. Refer to [Data compression settings](/docs/en/operations/server-configuration-parameters/settings#server-settings-compression) for more details.
```

Comment:
[...] and can only be used after setting server setting `enable_deflate_qpl_codec` to true.